Header header
geometry_msgs/Point prev
geometry_msgs/Point ctrl
geometry_msgs/Point next
float64 duration
float64 max_acc
float64 acc_per_err